bin/update_crds.sh

```diff
@@ -29,7 +29,8 @@ SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOTDIR=$(dirname "${SCRIPTPATH}")
 cd "${ROOTDIR}"
 
-REPO="github.com/istio/api"
+# REPO="github.com/istio/api"
+REPO="github.com/jewertow/api"
```

Comment:
This is only a temporary change and I will revert it once my pull request to the Istio API is merged.

Reply:
See my comments in the doc - I don't think an API change is required or best UX, the protocol, tunneling, etc can be auto-configured in the same way we do for auto-mtls..

Reply:
I think it's nice to have an option for explicitly configuring the behavior you want, especially for something like this.  I'm curious if there are any security considerations around automatically configure clients to use a specific tunneling protocol.  The same is true for auto-mtls, which allows for simplicity, but you still want a way to enforce mtls and prevent plain text requests (and you can by enforcing mtls and disabling auto-mtls).